VERSION

```diff
@@ -0,0 +1 @@
+0.1.0-dev
```

Comment:
```suggestion
v0.1.0-dev
```
Helps with the go mod vendoring. WIll make this change in the sample provider as well.